### DISCUSSION TITLE
全局不能测速一直不修啊

### DISCUSSION THREAD

**Discussion by 5lonely** [MetaCubeX/ClashMetaForAndroid#345] [General]
不能测速一直不修啊，有啥代替软件吗

**Comment by iwalke** [comment]
有个 [sing-box](https://play.google.com/store/apps/details?id=io.nekohasekai.sfa)，规则还不太懂，听说挺好用的。

**Reply by 5lonely** [reply]
不会呀

**Comment by 5lonely** [comment]


**Reply by Hunerk** [reply]
是啊，自动测速有些代理组也是点了一闪就没了。测不了速，BUG不修，新功能没有。就是更新个core就发出来了啥意思

**Comment by July250229** [comment]
搞得好像这群开源作者欠你们一样，先把礼貌学会再上网

**Reply by Hunerk** [reply]
> 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网
首先感谢作者的付出，其次你也别搞得开源就是上帝了，大家相互的。我们也没说什么出格的，测试版更新N次了，我们提的也不是什么不合适的问题。基础使用问题，一直不修复，问问而已。或者你说明什么原因。使用时urltest部分代理组点击测试一闪就没了，测试不了。我已使用flclash，建议楼主试试。CMFA作为备用

**Reply by July250229** [reply]
> > 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网
> > 搞得开源就是上帝了，大家相互的。我们也没说什么出格的，测试版更新N次了，我们提的也不是什么不合适的问题。基础使用问题，一直不修复，问问而已。或者你说明什么原因，我们也是关心，你自己看看更新内容就同步下核心。

你行你就上，提PR修，别在这里抱怨。开源跟你可不相互，你爱用就用，不爱用就自己修，而不是在这里抱怨作者一直不修。
还有你如果觉得你发的东西是关心，那我无话可说了。

**Reply by Hunerk** [reply]
> > > 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网
> > > 搞得开源就是上帝了，大家相互的。我们也没说什么出格的，测试版更新N次了，我们提的也不是什么不合适的问题。基础使用问题，一直不修复，问问而已。或者你说明什么原因，我们也是关心，你自己看看更新内容就同步下核心。
> 
> 你行你就上，提PR修，别在这里抱怨。开源跟你可不相互，你爱用就用，不爱用就自己修，而不是在这里抱怨作者一直不修。 还有你如果觉得你发的东西是关心，那我无话可说了。

我道你一声”不说了“。你自己玩吧。我不用了，我可以用脚投票，说都不能说。开源个毛线，自己搞着玩就行了，你嗨吧，

**Reply by qpst4** [reply]
> 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网

真相才是最快的刀这么急哈哈, 标题=客观事实, 我没看出他们有什么不礼貌. 看看我发的discussion那才叫不礼貌

**Reply by July250229** [reply]
> > 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网
> 
> 真相才是最快的刀这么急哈哈, 标题=客观事实, 我没看出他们有什么不礼貌. 看看我发的discussion那才叫不礼貌

一路货色，找什么存在感🤭 . 只怕是没人来提PR修你们用不上，急得不行.

**Reply by qpst4** [reply]
> > > 搞得好像这群开源作者欠你们一样，先把礼貌学会再上网
> > 
> > 
> > 真相才是最快的刀这么急哈哈, 标题=客观事实, 我没看出他们有什么不礼貌. 看看我发的discussion那才叫不礼貌
> 
> 一路货色，找什么存在感🤭 . 只怕是没人来提PR修你们用不上，急得不行.

用不上? 早就不用了🤣👉

**Reply by July250229** [reply]
不用了还来阴阳怪气，你不是贱是什么？爹妈教的好😜

**Comment by fouvromenth** [comment]
主要是Android缺开发者，现在以能用就行为主

项目整个是fork的旧版clash for android

有部分接口针对的是当时不开源的P核

甚至新版的clash for android是完全不开源的，这边都没fork到

之后clash删库，android直接停滞archive了

差不多两个月在强烈要求+帮助下改了核心库重新编译，期间Android版是没更新过的

可以查commit记录，去年11月到现在的更新，都是本着是能用就行，保持内核同步

内核开发者自己都推荐android用第三方的，关于android的issue除灾难性bug都不管

所以现在就是 PR is Welcome

**Comment by cutetool** [comment]
可以用surfboard试试

**Comment by Hunerk** [comment]
> 不能测速一直不修啊，有啥代替软件吗

flclash，

**Reply by gsxgmpkt9020** [reply]
救苦救难观世音菩萨！

**Reply by 5lonely** [reply]
同问

**Comment by jijiqw** [comment]
看这评论区本仓库是没啥希望了, 建议都去flclash, 我下了体验丝滑
